pest/examples/parens.rs

```diff
@@ -1,9 +1,8 @@
 extern crate pest;
 
 use std::io::{self, Write};
-use std::rc::Rc;
 
-use pest::inputs::{Input, Position};
+use pest::inputs::Position;
```

Comment:
I guess it would be more appropriate to rename this to `input` now. Maybe even something else.

Reply:
I'm not sure I see `Position` and `Span` as so related any more. The former does matching operations and transformations; the latter is more of a useful container. I think I'll pull them out of a module, as there's not really one concept that binds them together.